storage/plugins/discogs/DiscogsPlugin.php

```
@@ -50,17 +50,136 @@ public function activate(): void
+     * rawurlencode() when composing the URL.
+     */
+    public static function buildMusicBrainzQuery(string $input): string
+    {
+        $kind = self::identifierKind($input);
+        $field = $kind === 'catno' ? 'catno' : 'barcode';
+        if ($kind === 'catno') {
+            $value = '"' . addcslashes($input, '"\\') . '"';
+        } else {
+            $digits = preg_replace('/\D+/', '', $input);
+            $value = ($digits !== null && $digits !== '') ? $digits : $input;
+        }
```

PHPMD (2.15.0)

[error] The method buildMusicBrainzQuery uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)